bestiary/core/schema.py

```diff
@@ -108,6 +113,21 @@ class Meta:
         description = "Transaction objects representing atomic sets of operations modifying the database."
 
 
+class EcosystemType(DjangoObjectType):
+    """Type for Ecosystem objects which represent an abstract set of projects as defined in models"""
```

Comment:
I think the documentation should be useful for the user of the GraphQL API. This description seems to be more related for a developer.

Reply:
This description is limited to one line as far as I tried, so this is my proposal: 

```
Ecosystem objects are meant to gather a set of projects sharing a common context
```

WDYT?